rekor/rekor.go

```diff
@@ -269,24 +269,24 @@ func ParseEnvelopeFromEntry(entry *models.LogEntryAnon) (dsse.Envelope, error) {
 		return env, fmt.Errorf("couldn't parse rekor entry: %w", err)
 	}
 
-	baseModel := models.Dsse{}
+	baseModel := models.Intoto{}
 	if err := baseModel.UnmarshalJSON(decodedBody); err != nil {
 		return env, fmt.Errorf("failed to parse rekor entry: %w", err)
 	}
 
-	eimpl, err := types.NewEntry(pe)
+	eimpl, err := types.CreateVersionedEntry(pe)
```

Comment:
Newer versions of rekor removed `NewEntry`. As best I can tell the `CreateVersionedEntry` is essentially the same in the code so I used that instead.